docs/spec/nmt-wrapper.md

```diff
@@ -0,0 +1,83 @@
+# Namespaced Merkle Tree Wrapper
+## Abstract
+In Celestia, block transactions are grouped into identical-size shares and arranged in a `k` by `k` matrix, called [original data square](https://github.com/celestiaorg/celestia-app/blob/specs-staging/specs/src/specs/data_structures.md#arranging-available-data-into-shares).
```

Comment:
[optional] I expect this link to break when we merge the specs-staging branch to main and delete the branch. To avoid the broken link we can use permalinks. 

```suggestion
In Celestia, block transactions are grouped into identical-size shares and arranged in a `k` by `k` matrix, called [original data square](https://github.com/celestiaorg/celestia-app/blob/c09843d07d4c3842753138de96b304b4866e8f5d/specs/src/specs/data_structures.md?plain=1#L494).
```

Reply:
Good point, I fixed all the links accordingly. Please see https://github.com/celestiaorg/celestia-app/pull/1442/commits/22e737c961c551e85166cfd9ecb455d248c5d8c4.